### ISSUE TITLE
VSCode Causes Windows 10 to Freeze and Crash

### ISSUE THREAD

**Issue by themightyjohn** [Microsoft/vscode#21556]
- VSCode Version: 1.9.1
- OS Version: Windows 10 Enterprise x64
- Graphics Cards: Intel HD Graphics 530 (20.19.15.4531) / Nvidia Quadro M2000M (21.21.13.6949)
- Laptop: Dell Precision 7510 - Latest drivers for everything (as of 28/02/2017)
-  i7-6820HQ - 64GB RAM - 512GB Toshiba NVMe - Intel Display Audio + Realtek Audio

Steps to Reproduce:

1. Open VSCode
2. Let or put laptop to sleep / power save.
3. Wake laptop up and start using.
3. After a short amount of time, the entire Windows GUI stops responding after a few more click the mouse freezes.

No ability to recover from this except a hard power cycle.

Windows outputs no errors in the Windows event logs.

This happens every time with VS code. It does not happen with anything else we have running. 

The same thing on multiple laptops of the same spec.

Compatibility mode (Win 7 / 8 ) does not help.

Not sure exactly where the problem lies (Windows 10 / Intel Graphics / Nvidia Graphics / VSCode) but as its only VSCode that does it, i'm starting by posting it here.

Thanks for any help/pointers or fixes.

If you need more info or logs please ask.

Extensions:

abusaidm.html-snippets-0.1.0
christian-kohler.npm-intellisense-1.3.0
christian-kohler.path-intellisense-1.2.0
dbaeumer.vscode-eslint-1.2.6
donjayamanne.bower-0.0.3
eg2.tslint-0.8.1
fknop.vscode-npm-3.3.0
msjsdiag.debugger-for-chrome-2.5.5
pflannery.vscode-versionlens-0.12.1
robertohuertasm.vscode-icons-7.3.0
robinbentley.sass-indented-1.3.0
seanmcbreen.Spell-0.9.1
wayou.vscode-todo-highlight-0.4.0
Zignd.html-css-class-completion-1.0.3

**Comment by bpasero** [comment]
Does it reproduce when you run `code --disable-gpu`

**Comment by themightyjohn** [comment]
Typically after weeks of non-stop crashing, I cant replicate the issue anymore?! 

- Either the Intel (from the Intel website rather then Dell website) drivers i updated today (Bluetooth and Wireless) fixed a bug.
- I updated some of the extensions and maybe one fixed the bug.
- I also ran the Windows 10 compatibility troubleshooter which put it into Win 8 mode, iv since unchecked the mode to test but the wizard might have changed something else as well. 
- Or i'm just not hitting the bug at the moment. 

I haven't tried the switch you supplied, but if it resurfaces i will test it out. 

Thanks for the suggestion. 

Closing the issue for the time being.
